package-support-policy.md

```diff
@@ -0,0 +1,90 @@
+# Support Policy for .NET Packages
+
+NuGet packages owned by the .NET team are supported as long as they target at least one framework that is still actively supported.
+
+The minimum supported .NET versions are:
+
+- .NET Framework 4.6.2
+- .NET Standard 2.0
+- .NET 6.0
+
+The support policy of our frameworks can be found here: https://dotnet.microsoft.com/en-us/platform/support/policy
+
+
+## Package and project compatibility
+
+A package targeting any of these above mentioned supported frameworks is expected to be compatible with projects targeting any newer versions. For example:
+
+- If you have a project that targets .NET 8.0, and it consumes a package that targets either .NET Standard 2.0 or .NET 6.0, you should expect the package to be compatible with your project because .NET Standard 2.0 supports .NET 8.0 and .NET 8.0 is greater than .NET 6.0.
+- If you have a project that targets .NET 6.0, and it consumes a package that targets .NET Standard 2.0 and .NET 8.0, you should expect the package to be compatible with the project because .NET Standard 2.0 supports .NET 6.0.
+- If you have a project that targets .NET 7.0, and it consumes a package that targets .NET 6.0, you should expect the package to be compatible with the project because .NET 7.0 is greater than .NET 6.0.
+- If you have a project that targets .NET Framework 4.8, and it consumes a package that targets .NET Standard 2.0 and .NET Framework 4.6.2, you should expect the package to be compatible with the project because .NET Standard 2.0 supports .NET Framework 4.8 and .NET Framework 4.8 is greater than .NET Framework 4.6.2.
+- If you have a project that targets .NET 6.0 and it consumes a package that targets .NET 8.0, the package is not compatible with the project.
+- If you have a project that targets .NET Framework 4.6.2 and it consumes a package that targets .NET Framework 4.8, the package is not compatible with the project.
+
+The detailed list of .NET Frameworks supported by each .NET Standard version can be found here: https://learn.microsoft.com/en-us/dotnet/standard/net-standard
+
+## Package types
```

Comment:
A bunch of this content should be in docs. It isn't "support information".

Reply:
Where in docs would you suggest putting it? Or would this information be more useful if it was rewritten with a more precise/specific focus on support?

I thought this information was relevant to users from the perspective of the lifecycle of a package because it might be difficult to understand for cases where it's not clear to them what happened to (or where is) the latest version of the package.

I'll give you an example: Why is MS Docs showing new APIs for an assembly that I've been using as a NuGet package, but the latest version of said package does not include those APIs? Surprise surprise: turns out the assembly got absorbed by the shared framework and the latest version contains those new APIs.

Reply:
I don't know where the library teams hosts general content in docs about packages. If there isn't such a place, now might be a good time to create one. Major point is that this information goes well beyond support.

> I'll give you an example: Why is MS Docs showing new APIs for an assembly that I've been using as a NuGet package, but the latest version of said package does not include those APIs? Surprise surprise: turns out the assembly got absorbed by the shared framework and the latest version contains those new APIs.

Sure. Let's separate out lifecycle (when a package goes EOL) from weird migration topics. They have a relationship, but are totally separate concerns. Support documents are intended to be super crisp and as terse as possible (while still readable/useful).